spec/support/newflow_feature_helpers.rb

```diff
@@ -139,3 +140,36 @@ def generate_expired_login_token_for_user(user)
 def expect_reauthenticate_form_page
   expect(page).to have_content(t :"login_signup_form.login_page_header")
 end
+
+# Call this method with a block to test login/signup with a social network
+def simulate_login_signup_with_social(options={})
+  options[:name] ||= 'Elon Musk'
+  options[:nickname] ||= 'elonmusk'
+  options[:uid] ||= '1234UID'
+
+  begin
```

Comment:
not a big deal since this is a test support method...but this method violates SRP..it's doing several things:  initializing options, setting a OmniAuth identity, and setting OmiAuth providers.   Consider breaking these things into at least 2 other methods (the identify and providers parts).  you can always use the "squint test" to see the shape of a method and take good guesses of how it could be broken down a bit more.